icons/lasso-select.svg

```diff
@@ -0,0 +1 @@
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><defs><style>.cls-1,.cls-2{fill:none;stroke:#000;stroke-miterlimit:10;stroke-width:2px;}.cls-1{stroke-linecap:round;}</style></defs><path class="cls-1" d="M7,22a5,5,0,0,1-2-4"/><path class="cls-2" d="M14,17.84A13.06,13.06,0,0,1,12,18a11.93,11.93,0,0,1-5-1.07"/><path class="cls-2" d="M3.34,14A6.8,6.8,0,0,1,2,10C2,5.58,6.48,2,12,2s10,3.58,10,8a7.19,7.19,0,0,1-2.6,5.38"/><circle class="cls-2" cx="5" cy="16" r="2"/><path class="cls-2" d="M14.33,22h-.09a.35.35,0,0,1-.24-.32v-10a.34.34,0,0,1,.33-.34.32.32,0,0,1,.21.08l7.34,6a.33.33,0,0,1-.21.59H17.18l-2.57,3.85A.35.35,0,0,1,14.33,22Z"/></svg>
```

Comment:
With 2px padding between elements
<img src="https://user-images.githubusercontent.com/11825403/92466077-d8708b00-f1cf-11ea-892c-aae6417b72aa.png" width="240" /> 

```suggestion
<svg 
  xmlns="http://www.w3.org/2000/svg" 
  width="24"
  height="24"
  viewBox="0 0 24 24"
  fill="none"
  stroke="currentColor"
  stroke-width="2"
  stroke-linecap="round"
  stroke-linejoin="round"
>
  <path d="M7 22a5 5 0 01-2-4" />
  <path d="M7 16.93c.96.43 1.96.74 2.99.91" />
  <path d="M3.34 14A6.8 6.8 0 012 10c0-4.42 4.48-8 10-8s10 3.58 10 8a7.19 7.19 0 01-.33 2" />
  <path d="M5 18a2 2 0 100-4 2 2 0 000 4z" />
  <path d="M14.33 22h-.09a.35.35 0 01-.24-.32v-10a.34.34 0 01.33-.34c.08 0 .15.03.21.08l7.34 6a.33.33 0 01-.21.59h-4.49l-2.57 3.85a.35.35 0 01-.28.14v0z" />
</svg>
```

Reply:
using the same cursor from the suggested lasso+cursor
![Screen Shot 2020-09-08 at 8 52 11 AM](https://user-images.githubusercontent.com/41239783/92478732-98e67680-f1b0-11ea-90ab-94f97dd0db07.png)

to get 2 pixels spacing the bottom line becomes a circle, looks wrong to me, but I'm not sure, so here's that for comparison
```suggestion
<svg
  xmlns="http://www.w3.org/2000/svg"
  width="24"
  height="24"
  viewBox="0 0 24 24"
  fill="none"
  stroke="currentColor"
  stroke-width="2"
  stroke-linecap="round"
  stroke-linejoin="round"
>
  <rect rx="1" height="4" width="4" y="2" x="2" />
  <rect rx="1" height="4" width="4" y="2" x="18" />
  <rect rx="1" height="4" width="4" y="18" x="2" />
  <line y2="14" x2="4" y1="10" x1="4" />
  <line y2="12" x2="20" y1="10" x1="20" />
  <line y2="4" x2="10" y1="4" x1="14" />
  <line y2="20" x2="10" y1="20" x1="10" />
  <path d="M14.33 22h-.09a.35.35 0 01-.24-.32v-10a.34.34 0 01.33-.34c.08 0 .15.03.21.08l7.34 6a.33.33 0 01-.21.59h-4.49l-2.57 3.85a.35.35 0 01-.28.14v0z" />
</svg>
```